src/pages/Community/CommunityImgItem.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import * as C from './CommunityStyle';
+
+export default function CommunityImgItem({ img, handlePlusItem }) {
+  return (
+    <>
+      <C.CommunityImg
+        key={img.id + 1}
+        src={img.img}
+        alt="img"
+        onClick={handlePlusItem}
+      />
+    </>
```

Comment:
```suggestion
      <C.CommunityImg
        key={img.id + 1}
        src={img.img}
        alt="img"
        onClick={handlePlusItem}
      />
```
꼭 fragment로 감싸지 않아도 될 것 같아요!